packages/ds-medicare-gov/src/styles/components/_override.help_drawer.scss

```diff
@@ -2,31 +2,15 @@
 $mct-drawer-close-button-size: 26px;
 $mct-drawer-heading-font-size: $h1-mobile-font-size;
 
-.mct-c-help-drawer {
```

Comment:
Since mgov styles are approved by Melissa, I removed this wrapping class and updated tokens/styles where appropriate.